src/extension/app/aem-sidekick.css.js

```diff
@@ -44,6 +44,7 @@ export const style = css`
     width: auto;
     min-width: 640px;
     max-width: var(--sidekick-max-width);
+    white-space:nowrap;
```

Comment:
We can't do it like this. It will prevent longer error messages from wrapping. Maybe it can be set at this level and all wrapping in the toast container.

![Screenshot 2025-01-20 at 10 33 14 AM](https://github.com/user-attachments/assets/b936085e-28ee-4ad0-851d-84ca15771c58)

Reply:
Thanks @dylandepass  .
modified 
<img width="358" alt="image" src="https://github.com/user-attachments/assets/2abc1e95-7bb7-414e-a838-e0debc944c62" />

with which the messages are wrapping
<img width="897" alt="image" src="https://github.com/user-attachments/assets/aefa711d-38bb-403b-aab2-ca55a7585bf2" />

Reply:
please review